aztk/spark/models/plugins/resource_monitor/readme.md

```diff
@@ -0,0 +1,60 @@
+# Using the Resrouce Monitor Plugin
+
+The resource monitor plugin is useful for tracking performance counters on the cluster. These include counters such as Percent CPU used per core, Disk Read, Disk Write, Network In, Network out, and several others. Simply enabling the plugin in your cluster.yaml will deploy all the necessary components to start tracking metrics.
+
+This plugin takes advanage of the TICK monitoring stack. For more information please visit the [influx data](https://www.influxdata.com/time-series-platform/) web page.
```

Comment:
I feel like we should put something here about how to persist the data to storage.